xep-0402.xml

```diff
@@ -102,25 +115,42 @@
 <conference xmlns=']]>&namespace;<![CDATA['
             name='Council of Oberon'
             autojoin='true'>
-    <nick>Puck</nick>
+  <nick>Puck</nick>
 </conference>
 ]]></example>
         <p>This bookmark would be displayed as 'Council of Oberon' and, if activated, would attempt to join the conference room 'council@conference.underhill.org' with nickname 'Puck'.</p>
         <p>Note that a bookmark item MUST contain only one conference room.</p>
-        <p>Note also that a conference element has no truly mandatory attributes or child elements, though a name SHOULD be given. Thus the following is legal:</p>
+        <p>Note also that a conference element has no truly mandatory attributes or child elements, thus the following is legal:</p>
         <example caption='Minimalist conference element'><![CDATA[
 <conference xmlns=']]>&namespace;<![CDATA['/>
 ]]></example>
     </section1>
 
-    <section1 topic='Examples' anchor='examples'>
+    <section1 topic='Workflow' anchor='workflow'>
+
+        <section2 topic='Registering to notifications' anchor='enable-updates'>
+            <p>A client interested in bookmarks SHOULD include the 'urn:xmpp:bookmarks:0+notify' feature in its &xep0115;, as per &xep0163;, so that it receives notifications for updates done by other clients of the user, and react accordingly.  The actual notifications are explained in the <link url='#notifications'>Bookmark Notifications</link> section of this specification.</p>
+
+            <example caption='Client replies with +notify to a XEP-0030 query from its server'><![CDATA[
+<iq from='juliet@capulet.lit/balcony' to='capulet.lit' type='result' id='disco1'>
+  <query xmlns='http://jabber.org/protocol/disco#info'>
+    <identity category='client' type='pc' name='poezio'/>
```

Comment:
I don't think we should put subtle advertisements for clients in the XEPs.

Reply:
Should I invent a non-existing client instead?  I remember seeing mentions of Psi or Swift in some XEPs, so I thought using an existing client was something acceptable, especially since otherwise the XEP-0030 answer is invalid when there is no identity.

Reply:
I guess it's ok.  My first impression is against it but I'm not going to argue over it.